common/map_test.cpp

```diff
@@ -303,13 +299,13 @@ TYPED_TEST(MapTest, GrowForInsert) {
     SCOPED_TRACE(llvm::formatv("Key: {0}", i).str());
     ASSERT_TRUE(m.Erase(i));
   }
-  m.GrowForInsertCount(1717);
-  for (int i : llvm::seq(128, 1717 + 128)) {
+  m.GrowForInsertCount(321);
```

Comment:
The before and after numbers here seem a bit magical / arbitrary. Where do 1717 and 321 come from?

Reply:
Just arbitrary numbers that don't exactly match a power of two (or 7/8ths of a power of 2) and are larger than the current table size.